tests/integration/simulacron/test_endpoint.py

```diff
@@ -27,23 +27,24 @@
 @total_ordering
 class AddressEndPoint(EndPoint):
 
-    def __init__(self, address):
+    def __init__(self, address, port=9042):
         self._address = address
+        self._port = port
 
     @property
     def address(self):
         return self._address
 
     @property
     def port(self):
-        return None
+        return self._port
```

Comment:
port can't be None when using Twisted:
https://github.com/datastax/python-driver/blob/master/cassandra/io/twistedreactor.py#L269